### ISSUE TITLE
Unexpected behavior of s_attributes() for GERMAPARLMINI

### ISSUE THREAD

**Issue by ChristophLeonhardt** [PolMine/polmineR#245]
In the most recent GERMAPARLMINI sample corpus, I noticed that `s_attributes()` behaves unexpectedly when applied to the entire corpus and the structural attribute „protocol_lp“ is included. This works fine for subcorpora in general and for corpora when protocol_lp is not part of the structural attributes. Maybe issue #244 is relevant here, too.

So, this is what I have done:

```
s_attributes("GERMAPARLMINI", s_attribute = c("speaker", "party", "protocol_lp"))
```

What I would expect is a data.table with the columns „speaker“, „party“ and „protocol_lp“. But the result of this call differs in two ways:

* The first attribute named in the `s_attribute` argument is represented with the column name „value“ in the resulting data.table. In the example above, the columns would be named „value“, „party“ and „protocol_lp“
* There are additional columns in the resulting data.table: „struc“, „cpos_left“, „cpos_right“. 

The misnamed column ("value" instead of "speaker") is probably a bug? The additional columns, however, can be really useful, but I think that in a lot of scenarios these are not necessary and that the data.table is not supposed to look like that by default, especially when the `unique` argument in the call defaults to TRUE.

Interestingly, both of these issues do not occur for subcorpora regardless of the s-attributes defined.

```
corpus("GERMAPARLMINI") |>
  subset(date == "2009-11-12") |>
  s_attributes(c("speaker", "party", "protocol_lp"))
```

This is what I would expect the data.table that is returned to look like for corpora, too.

So in conclusion, I think that an option to add these additional columns would be useful for both corpora and subcorpora but by default, these should not be part of the data.table.
